app/models/import.rb

```diff
@@ -0,0 +1,16 @@
+class Import
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  validates :file, presence: true
+
+  attr_accessor :file
+
+  def initialize(file = nil)
+    @file = file
+  end
+
+  def save
+    # TODO
```

Comment:
maybe think about logic to check we haven't lost any records - maybe check sizes between new upload and existing records to protect against uploading partial files. Maybe add a 'skip checks' option to be used on initial seed ? or initial seed only from rake task ?